### ISSUE TITLE
请求加上耗材余量提示功能

### ISSUE THREAD

**Issue by wgqi1126** [bambulab/BambuStudio#5060]
通过每次打印消耗的耗材，来计算剩余多少耗材。

**Comment by huxiaoyi0625** [comment]
I agree with you! It's a great trouble when the filament is exhausted. Especially, when there are no auxiliary filament in the AMS, I have to give up printing.
